website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -0,0 +1,353 @@
+  3. The NGINX Ingress Controller should be enabled by default. If not, [install ingress controller on Google Kubernetes Engine](https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke)[GKE]. Ensure that you enable the correct version for your cluster.
+
+</TabItem>
+<TabItem label="AWS EKS" value="AWSEKS">
+
+   1. Set up the Kubernetes cluster by [creating a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
+    2. Ensure that you have a default storage class running. If not, [create default storage class](https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html)
+    3. The NGINX Ingress Controller should be enabled by default. If not, [install NGINX Controller for AWS EKS](https://kubernetes.github.io/ingress-nginx/deploy/#network-load-balancer-nlb).
+    Ensure that you enable the correct version for your cluster.
+
+</TabItem>
+<TabItem label="Microk8s" value="Microk8s">
+
+  1. Set up Kubernetes by following [working with kubectl](https://microk8s.io/docs/working-with-kubectl).
+  2. Ensure that you have a default storage class running. If not, [enable storage class](https://microk8s.io/docs/command-reference#heading--microk8s-enable).
+  3. The NGINX Ingress Controller should be enabled by default. If not, [add on: Ingress](https://microk8s.io/docs/addon-ingress). Ensure that you enable the correct version for your cluster.
+    
+</TabItem>
+</Tabs>
+
+## Install Appsmith
+
+You've installed and configured the Kubernetes cluster. Now, you can install Appsmith using Helm.
+
+1. Download the [`values.yaml`](https://bit.ly/3ETEgPT) file, and place it into the Appsmith installation folder.
+
+OR, run the following cURL:
+
+```bash
+curl -L https://bit.ly/3ETEgPT -o $PWD/values.yml
+```
+
+2. Add Appsmith into your repository using Helm. First, add the Appsmith Helm repository to your local Helm installation. It allows Helm to download the Appsmith chart from the Appsmith GitHub repository.
+
+
+```bash
+helm repo add appsmith https://helm.appsmith.com
+```
+3. Update the Helm repository to make sure you have the latest version of the Appsmith chart by running the following command:
+
+```bash
+helm repo update
+```
+4. Install Appsmith on your Kubernetes cluster using the Helm chart. Replace `<your-namespace>` with the namespace you want to install Appsmith in, and `<release-name>` with a unique name for the Appsmith release:
+
+```bash
+helm install <release-name> appsmith/appsmith --namespace <your-namespace>
+```
+
+It creates all the necessary Kubernetes objects to run Appsmith on your clusters, such as pods, services, and ingresses with the default configuration. Read more on configurations you can do in the [Configure Appsmith](#configure-appsmith) section.
+
+5. Wait for Appsmith to be ready by running the following command:
+
+```bash
+kubectl -n <your-namespace> rollout status deployment <release-name>-appsmith
```

Comment:
We'll have to check the technical correctness for this.

Reply:
@sharat87 - please confirm.

Reply:
This command doesn't _wait_ for the cluster to be ready, it just displays the status. It's not strictly necessary to be included here, but if we want to include it, we need to frame it like "You can check the status of your installation with this" or something of that sort.